frontend/src/components/common/icon/icon.tsx

```diff
@@ -0,0 +1,39 @@
+import { ReactComponent as BillingIcon } from 'assets/icons/billing.svg';
+import { ReactComponent as CourseIcon } from 'assets/icons/course.svg';
+import { ReactComponent as EducationIcon } from 'assets/icons/education.svg';
+import { ReactComponent as HomeIcon } from 'assets/icons/home.svg';
+import { ReactComponent as MentorsIcon } from 'assets/icons/mentors.svg';
+import { ReactComponent as SettingsIcon } from 'assets/icons/settings.svg';
+import { FC, IconName, SVGProps } from 'common/types/types';
+import { getValidClasses } from 'helpers/helpers';
+
+import styles from './styles.module.scss';
+
+const icons: Record<
+  IconName,
+  FC<
+    SVGProps<SVGSVGElement> & {
+      title?: string | undefined;
```

Comment:
`?` already means that value could be undefined
```suggestion
      title?: string;
```